falcon/response.py

```diff
@@ -771,6 +776,19 @@ def add_link(self, target, rel, title=None, title_star=None,
         if anchor is not None:
             value += '; anchor="' + uri_encode(anchor) + '"'
 
+        if crossorigin is not None:
+            crossorigin = crossorigin.lower()
```

Comment:
Note that I added support for case-insensitive `crossorigin` values.

The `Link` header itself seems to be quite poorly documented, however the available documentation states that it should be treated exactly in the same way as the corresponding `<link>` HTML5 element [`crossorigin` attribute](https://www.w3.org/TR/html50/document-metadata.html#attr-link-crossorigin). Which in turn is an [enumerated attribute](https://www.w3.org/TR/html50/infrastructure.html#enumerated-attribute), hence [ASCII case-insensitive](https://www.w3.org/TR/html50/infrastructure.html#ascii-case-insensitive).

Reply:
That is correct, it is treated by the user agent the same as the `<link>` element, although some browser versions support certain features only when expressed as an element vs. a header.